src/selectedtests/datasource/datasource_cli.py

```diff
@@ -37,7 +32,8 @@ def cli(ctx, verbose: bool, mongo_uri: str):
     ctx.ensure_object(dict)
     ctx.obj["mongo"] = MongoWrapper.connect(mongo_uri)
 
-    _setup_logging(verbose)
+    verbosity = Verbosity.DEBUG if verbose else Verbosity.INFO
```

Comment:
[nit] In other repos, we have been using the click `count` option to set the verbosity (see [here](https://github.com/10gen/build-baron-tools/blob/42c9568632540bd676eef3e0bee78f4af75e7d88/src/bb/bb_cli.py#L143)). Instead of explicitly setting it to a value. The `count` option will set the parameter to the number of times the option was specified. So specifying it once would give INFO, twice DEBUG, etc.

Reply:
I saw that, I thought we could add that later if needed because right I've only found INFO and DEBUG useful. (The workers all use INFO level logging and I use DEBUG when running cli's locally.)